build/all-confirmed.yar

```diff
@@ -1,61 +1,61 @@
-  "6d284295cc9e09b828e970c6492ac44d689804bd": "Devinc_brouwmarkt/License.php",
-  "70e0e50bd58249f6167a158d129398f8c7661ba8": "VladimirPopov/WebForms/Block/Webforms.php",
-  "7766fa413892141f85460baf3df6b24f3de3c2c4": "Devinc/License/Block/Adminhtml/System/Config/Form/Select.php",
-  "80b2545f16ab1c5e94d2555aac655d36a7d884c9": "SM/XPos/Helper/License.php",
-  "86347e089a600e097ba29082ea9e0d76bda791cd": "Devinc/License/Model/License.php",
-  "8d70d65ac7f25232572f4ad418ad02e759931bb4": "Devinc_brouwmarkt/Head.php",
-  "8efa34f6fa2b7eb706ea6ea6e991e859659ed2eb": "Devinc_brouwmarkt/Data.php",
-  "904287dffb72287d91c0e705efd36641cb45e24f": "all4coding/license.js",
-  "90c2347adc2b4b21269f85c291cc6abe97112ea7": "Devinc/License/Block/Adminhtml/System/Config/Form/Obscure.php",
-  "914d0a7f11c2d434801c8f5f3c42e9bd2e9bd32d": "Devinc_brouwmarkt/Select.php",
-  "95e161fd802fa66554cbde5016ec80c2ecb09cfa": "Devinc_brouwmarkt/Url.php",
-  "98b1f9558397f6bd5823206d49d2c4c892f29aad": "Devinc_brouwmarkt/Navigation.php",
-  "9df90c17d8fe44c94bec7293562087e5c8772dbf": "Devinc/License/Model/System/Config/Backend/License.php",
-  "a146b644658f7c66c364f40068ae48f0657b599d": "Magegiant_GiantPoints_Model_Mysql4_Abstract.php",
-  "a50e0fb45c5b1cc6dbfcb77f2d731567679e24f9": "Devinc/License/controllers/IndexController.php",
-  "a54d08a17ada4e5a066187352e30fdbb0095a2dc": "VladimirPopov/WebForms/Block/Adminhtml/Results/Grid.php",
-  "b17bb4e11fe680b39a64b05b71f33600f773cdf4": "Devinc/License/Block/Gomobile/Page/Html/Head.php",
-  "b76e9601bf986430e269ee1e97387f157c84e9ba": "NeoPI/animal_shell_poc.php",
-  "baac35ffb7c99a62bb269a24a83bb59d92240c8a": "Devinc/License/Block/Adminhtml/System/Config/Form/Textarea.php",
-  "bd572a0531e9dfa8b77bd3869312d1e2bb0ac749": "Devinc_brouwmarkt/files",
-  "bfe98572af1a0e2f8091a5ef5a12d99e84cfa567": "GoMage/Checkout/Helper/Data.php",
-  "c6046941de9ffd9362a20df3ed8590e5d9ae0f16": "VladimirPopov/WebForms/controllers/Adminhtml/Webforms/FieldsController.php",
-  "c618da41265067743bf88c07980b7f83791d7f3a": "Devinc_brouwmarkt/Mobiledetect.php",
-  "cfe0ca135dff5858b80a785f6f9bae7796bf791c": "Devinc/License/Block/Adminhtml/System/Config/Form/Multiline.php",
-  "d76d8b9fd79e3c70c678091cd6818cfe74e61e0d": "Devinc_brouwmarkt/Status.php",
-  "e25fb78d0b8deb62435df210ce67cdd1f36c3f75": "Devinc_brouwmarkt/Multiline.php",
-  "e2f1d7bfe010883b89f8efa7b30ee4b6c2e0f677": "Devinc/License/Block/Adminhtml/System/Config/Form/Multiselect.php",
-  "e3a1402618f86e61a2f90d2dced2ad0fe8cceb27": "GoMage/Navigation/Helper/Data__v4.0.php",
-  "e8f50f62cd7d8096903134912704a7907baaa7ef": "Devinc/License/Block/Adminhtml/System/Config/Form/Label.php",
-  "ea85a187ac096ebda3c216dac0b36c04f598da02": "VladimirPopov/WebForms/Block/Adminhtml/Webforms.php",
-  "f0e9b11f4085085feb25fff71739c39f40dc2499": "Wexto/Exporter/controllers/IndexController.php",
-  "f44079f558051a341d0e3a5591f3e2dd6c29435c": "Devinc/License/Model/Gomobile/Core/Design/Package.php",
```

Comment:
I didn't explicitly in the way that I said to myself I want to use Py2... :P 

But it appears that my PATH for python appears points to v2.7 rather than v3 and I probably did just do `python validate_signatures.py` ....